solvers/safe.py

```diff
@@ -0,0 +1,56 @@
+from benchopt import BaseSolver, safe_import_context
+
+
+with safe_import_context() as import_ctx:
+    import numpy as np
+    from scipy import sparse
+    from slopescreening.solver.parameters import SlopeParameters
+    from slopescreening.solver.slope import slope_gp
+
+
+class Solver(BaseSolver):
+    name = "Safe"
```

Comment:
Maybe an explicit name would be more future-proof? Like PGD_safe_screening?

Reply:
Yes, good point. But how about just SafePGD?

Reply:
Or nevermind, let's go with your suggestion.